src/Microsoft.ML.Ensemble/OutputCombiners/MultiStacking.cs

```diff
@@ -57,7 +57,7 @@ public Arguments()
                     env => new Ova(env, new Ova.Arguments()
                     {
                         PredictorType = ComponentFactoryUtils.CreateFromFunction(
-                            e => new FastTreeBinaryClassificationTrainer(e, new FastTreeBinaryClassificationTrainer.Arguments()))
```

Comment:
>  e => new FastTreeBinaryClassificationTrain [](start = 26, length = 44)

this is temporary, until FastTree is an estimator.

Reply:
Hi @sfilipi, honestly I feel like this sort of thing really should *not* have a default. It seems to be along the same lines as the discussion in #682.

---
In reply to: [217222841](https://github.com/dotnet/machinelearning/pull/865#discussion_r217222841) [](ancestors = 217222841)